src/components/Alteration/Articles/Articles.vue

```diff
@@ -68,14 +68,15 @@ export default class Articles extends Mixins(CommonMixin) {
 
   /** Emits Have Changes event. */
   @Emit('haveChanges')
-  emitHaveChanges (haveChanges: boolean): void {}
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  protected emitHaveChanges (haveChanges: boolean): void {}
```

Comment:
I tried changing this to...
```
protected emitHaveChanges (haveChanges: boolean): boolean {
return haveChanges
```
... but the unit tests say the emitted event looks different.

Has anyone seen this before? (I decided to not refactor this so no additional testing is required.)

Reply:
I pulled the PR and ran the unit test your intended code `protected ... return haveChanges}` in my local environment. All of the unit tests passed, including `Articles.spec.ts`.

Reply:
I picked a bad example :)

Some tests verify the actual emitted value. The problem I saw had to do with emitting `[[1]]` vs `[[1],[1]]` or something like that -- a different component.

Anyway, I was trying to change the emitter function so we don't need the eslint exception but there is a difference so I didn't change it.

(I'm OK with some thing being beyond my knowledge, hah.)